packages/core/src/components/StepList/_StepList.docs.scss

```diff
@@ -46,21 +46,100 @@ Style guide: patterns.step-list.react
 /*
 Step object
 
-A `step` object contains all information needed to render the step, including
-text, progress, and routing information.
-
-Step object properties:
-- **id** (`string`) - Unique string representing the step. Will be passed to onStepLinkClick as second parameter
-- **href** (`string`) - URL or partial URL that routes to the step. Will be passed to onStepLinkClick as first parameter
-- **completed** (`bool`) - Whether the step has been completed
-- **started** (`bool`) - Whether the step has been started
-- **isNextStep** (`bool`) - Whether this is the next unstarted step
-- **title** (`string`) - Text to display as the step title
-- **description** (`string`) - Additional text to display under (only rendered for top-level steps)
-- **linkText** (`string`) - Alternative text for the link or button for this step. Will override the defaults
-- **onClick** (`func`) - onClick handler for this specific step's link/button
-- **steps** (`Step[]`) - Array of substeps
-
+<table class="ds-c-table">
+  <thead>
```

Comment:
Because of the way <StepList> is setup, we cannot simply use `<Step>` to show the step object documentation. In order to be consistent with other prop documentation, I converted the bulleted list to a table. Unfortunately, this needs to be updated manually and is not automatically generated from React prop documentation

Reply:
Looks nice 👍